### ISSUE TITLE
Suggestion: Rename /src/doc/trpl directory to /src/doc/book

### ISSUE THREAD

**Issue by echochamber** [rust-lang/rust#25223]
When browsing the source code in github, all the links in the old rust guide files 404 because there is no /src/doc/book directory 

Example : 
https://github.com/rust-lang/rust/blob/master/src/doc/guide-tasks.md contains a link to here https://github.com/rust-lang/rust/blob/master/src/doc/book/concurrency.html (which will 404).

These links work on the actual doc pages: http://doc.rust-lang.org/guide-tasks.html links to http://doc.rust-lang.org/book/concurrency.html. Renaming /src/doc/trpl to /src/doc/book would make all these links work in both locations without breaking links from blog posts and whatnot to http://doc.rust-lang.org/book and without breaking the links to different on http://doc.rust-lang.org/book itself. 

More of a convenience than anything, but at no real cost either.

**Comment by daboross** [comment]
Wouldn't all the links still 404 in github though because they are to `*.html` files instead of `*.md` files?

**Comment by steveklabnik** [comment]
Yes, as @daboross says, this wouldn't fix the linking issue.

I wouldn't mind someone writing a patch to change this. The reason that they're different is that i originally was going to make them both `trpl`, but we decided to switch to `book` at the last minute, and I only renamed the output, not the input.

That being said, I don't feel strongly enough about it to write said patch myself, so I'm going to close this. I'd accept a PR from someone else though.
